scripts/run-kani.sh

```diff
@@ -0,0 +1,206 @@
+#!/bin/bash
+
+set -e
+
+usage() {
+    echo "Usage: $0 [options] [-p <path>] [--kani-args <command arguments>]"
+    echo "Options:"
+    echo "  -h, --help         Show this help message"
+    echo "  -p, --path <path>  Optional: Specify a path to a copy of the std library. For example, if you want to run the script from an outside directory."
+    echo "  --kani-args  <command arguments to kani>  Optional: Arguments to pass to the command. Simply pass them in the same way you would to the Kani binary. This should be the last argument."
+    exit 1
+}
+
+# Initialize variables
+command_args=""
+path=""
+
+# Parse command line arguments
+# TODO: Improve parsing with getopts
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        -h|--help)
+            usage
+            ;;
+        -p|--path)
+            if [[ -n $2 ]]; then
+                path=$2
+                shift 2
+            else
+                echo "Error: Path argument is missing"
+                usage
+            fi
+            ;;
+        --kani-args)
+            shift
+            command_args="$@"
+            break
+            ;;
+        *)
+            break
+            ;;
+    esac
+done
+
+# Set working directory
+if [[ -n "$path" ]]; then
+    if [[ ! -d "$path" ]]; then
```

Comment:
I'm in favour of using `[[` as a bash built-in instead of `[` (aka `test`), but elsewhere in the file you've chosen the latter. I'd suggest to be consistent.